### ISSUE TITLE
Plan address setting is not working correctly

### ISSUE THREAD

**Issue by DaniGTA** [plan-player-analytics/Plan#3021]
### Describe the issue
When i reverse proxy my url `test.test/mc/stats` to plan -> `plan:8805/mc/stats` i get **404** when i access `test.test/mc/stats/players`

because plan expects `plan:8805/players`

okay than i can just proxy `test.test/mc/stats` to plan -> `plan:8805`

nope because ressource are under `mc/stats` and i get **404** on `/mc/stats/static/js/8197.0a897d05.chunk.js` for example because the reverse proxy is requesting `/static/js/8197.0a897d05.chunk.js` and plan expects `/mc/stats/static/js/8197.0a897d05.chunk.js`

and when i change the address in plan to test.test it request the ressource in web browser to `test.test/static/js/8197.0a897d05.chunk.js` and this url will be not proxied to plan.

### Possible solution

> _`mc/stats` is just a example from the adress setting in the plan config._

pls apply the address to 
`/players` -> `mc/stats/players ` 
`/server` -> `mc/stats/players ` 
`/v1` -> `mc/stats/players` 
`/player` -> `mc/stats/players `

if you need the urls `/players`  `/server`  `/v1`  `/player` just make both available `/v1` and `/mc/stats/v1`

### Plugin versions
5.5 build 2307

### Additional information
    Alternative_IP:
        # %port% is replaced automatically with Webserver.Port
        Address: test.test/mc/stats
        Enabled: true
        
#### Nginx config
 
```
location ^~ /mc/stats {
       proxy_pass plan:8805;
       proxy_set_header Host $host;
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header X-Forwarded-For $remote_addr;
}
```

**Comment by AuroraLS3** [comment]
There is a regression unit test against this which passes and uses following nginx config
```nginx
    location ~ /plan/(.*) {
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $remote_addr;
        proxy_pass http://localhost:9001/$1$is_args$args;
    }
```
```
http://{nginx}/plan
```

**Comment by DaniGTA** [comment]
Thats how it works on my side:
## Reverse Proxy settings
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/85f8dcda-3956-4f52-b5a3-37232bca18c6)
## User view
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/92a477bb-7840-4467-ba11-ef04f2f33bac)
## Server view on `/server/Server 1/overview`
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/50176c3a-ee4c-4b48-9518-b152dea31980)
## Server view on `mc/stats/server/Server 1/overview`
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/095906ea-e880-431f-9341-a636a1f0dbc2)
## Plan settings
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/b16bf464-851b-488f-9a40-96d2cd8290b3)

What did i do wrong here?

**Comment by AuroraLS3** [comment]
There's a / missing after stats in nginx config, should be /mc/stats/(.*)

**Comment by DaniGTA** [comment]
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/1282f647-d484-4906-b7b4-04d81157c416)
Plan doesnt like that

## New settings:

![image](https://github.com/plan-player-analytics/Plan/assets/20490672/7f853993-e829-4cad-80a3-f35afac60bab)

**Comment by DaniGTA** [comment]
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/0f6bd1b2-fcd6-410a-8631-969554b2d3bb)

## New settings
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/965460d6-3626-46b7-9d09-ee0eee197e25)

**Comment by AuroraLS3** [comment]
You can try renaming the server with the servername setting.

There's also an access log log to console setting under webserver settings that helps debug reverse-proxy issues. I think this 400 error may be coming from somewhere in-between or something is parsing the URI %20 into a space and not encoding it back before passing to Plan

**Comment by DaniGTA** [comment]
Im using plan a long time and i guess my db got broke because:
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/5c9dcb6d-963f-4df4-a549-8dabae449219)
It says i should go to the Server plan but 
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/ea770167-417e-40d9-8a49-3cc80e431423)

`Server 1` response differently

![image](https://github.com/plan-player-analytics/Plan/assets/20490672/488e38d5-a3f1-44c3-9532-b21e92b196cc)
(I never changed the server name)

**Comment by AuroraLS3** [comment]
If the server name is default it is changed to `Server {db id}` when it's displayed anywhere.

**Comment by AuroraLS3** [comment]
Yeah you can change it but `Plan` is the default value so it is treated differently

**Comment by DaniGTA** [comment]
I know, i changed it to `default` and i tried to access `https://myurl.de/mc/stats/server/default` but this url just redirected me to `https://myurl.de/mc/stats/server/Server 1` and the server still thinks after a restart that the name is Plan.

![image](https://github.com/plan-player-analytics/Plan/assets/20490672/bcd1599a-a5a4-4680-abf4-dadb98d63a2f)
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/76823f3a-aec5-478d-bbda-f976ade47f52)

**Comment by DaniGTA** [comment]
I guess i have a very special case that i cant fix easy on my side.

It would be very helpful if `/mc/stats/server/Server 1/overview` would accessible like `/server/Server 1/overview`
because this mixed handling is not easy to redirect.

## This is how it look when i access `/server/Server 1/overview`

You can see here that ressources are **404** and data urls like overview or v1 are **200** (that make sense because i said plan to listen on /mc/stats/)
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/0f40d988-2b03-4a4d-a79c-b2fdbeb98842)

## This is how it look when i access `/mc/stats/server/Server`
You can see here that ressources are **200** and data urls like overview are **404**
![image](https://github.com/plan-player-analytics/Plan/assets/20490672/ea3a6e91-ce00-4ddf-bac6-8426873d09c7)

Even if this is unsolvable thanks for trying to help with this problem!

**Comment by AuroraLS3** [comment]
If you cant get the subdirectory setup working I recommend using a subdomain based one, it's a lot easier to setup <https://github.com/plan-player-analytics/Plan/wiki/External-Webserver-use:-Reverse-proxy#nginx-reverse-proxy>
